app/presenters/content_items_csv_presenter.rb

```diff
@@ -40,6 +38,29 @@ def csv_rows
     end
   end
 
+  def filename
+    "content-data-export-from-%<from>s-to-%<to>s-from-%<org>s%<document_type>s.csv" % {
+      from: @date_range.from,
+      to: @date_range.to,
+      org: organisation_title.parameterize,
+      document_type: @document_type.present? ? "-in-#{@document_type}" : ''
```

Comment:
Should we make organisation and document type both use kebab case? e.g. news_story -> news-story

Reply:
Sure. Organisation will already, as I that's what `.parameterize` adds in place of spaces. I've added some code now to replace the underscores with dashes for the document type.